### PR TITLE
[ESLint] RE-provide Sider's recommended set of rules

### PR DESCRIPTION
**Pull request by sleeeto**
> Explain a summary, purpose, or background for this change.

Enable Sider's recommended ruleset of 'ESLint'

> Refer related issues or pull requests (e.g. `Close #123` or `None`).

None

> Check the following if needed.

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

**Comment by sleeeto** [comment]
I found NO deprecated rules in this recommended's.

File: images/eslint/sider_recommended_eslint.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Sider's recommended ruleset is based on eslint:recommended,
+# and following rules are additionally enabled/disabled.
+# See https://eslint.org/docs/rules/ for more details.
+# And, we generated the ruleset using ESLint 7.17, at 2021/04/05.
```
Comment (ybiquitous) [inline images/eslint/sider_recommended_eslint.yml:4]: [suggestion]

- How about using ISO format `yyyy-mm-dd`?
- Fix grammar: `at` -> `on`

```suggestion
# And, we generated the ruleset using ESLint 7.17, on 2021-04-05.
```

**Comment by sleeeto** [comment]
@ybiquitous I updated it.

**Comment by ybiquitous** [comment]
@sleeeto Why JSON? We should use JS...

File: images/eslint/sider_recommended_eslint.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+// Sider's recommended ruleset is based on eslint:recommended,
+// and following rules are additionally enabled/disabled.
+// See https://eslint.org/docs/rules/ for more details.
+// And, we generated the ruleset using ESLint 7.17, on 2021-04-05.
+module.exports = {
+  "root": true,
```
Comment (ybiquitous) [inline images/eslint/sider_recommended_eslint.js:6]: [suggestion] When considering a case that this config may be extended by other configs, this `root` seems needless. How about removing it?

```suggestion
```

Comment (sleeeto) [reply images/eslint/sider_recommended_eslint.js]: I removed it.

**Review by ybiquitous — APPROVED** [review]
LGTM 👍 

Please rebase `master` before the merge.

**Review by yykamei — APPROVED** [review]
LGTM :+1:

**Comment by sleeeto** [comment]
@ybiquitous Ready.

**Pull request merged**